azkaban-common/src/main/java/azkaban/trigger/TriggerManager.java

```diff
@@ -115,6 +115,7 @@ protected ActionTypeLoader getActionLoader() {
   }
 
   public void insertTrigger(Trigger t) throws TriggerManagerException {
+    logger.info("Inserting trigger " + t + " in TriggerManager");
```

Comment:
Why do we need this? I believe we have some other logger info messages during inserting.